docs/best-practices.md

```diff
@@ -0,0 +1,49 @@
+### How to select good environmental variable name?
```

Comment:
Consider using markdownlint. 

It enforces some basic standards for markdown docs. 

For example, the file name is Best Practices, but there's no title to suggest the similar.